forums/settings.py

```diff
@@ -1,6 +1,6 @@
-#Custom settings
+# Custom settings
 from os.path import *
-#from config import *
+# from config import *
```

Comment:
Is this needed? Please confirm and remove. One way to confirm if imports are required is to check if there are any import errors when running your Django server